tests/components/google_drive/conftest.py

```diff
@@ -0,0 +1,25 @@
+"""PyTest fixtures and test helpers."""
+
+import pytest
+
+from homeassistant.components.application_credentials import (
+    ClientCredential,
+    async_import_client_credential,
+)
+from homeassistant.components.google_drive.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+CLIENT_ID = "1234"
+CLIENT_SECRET = "5678"
+
+
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture(autouse=True)
```
You will most likely neverhave a moment where you don't need them, so you can remove all the extra fixtures from the tests